lib/moleculer/transporters/redis.rb

```diff
@@ -163,13 +165,21 @@ def subscribe(channel, &block)
             channel: channel,
             block:   block,
             config:  @config,
-          ).connect
+          )
+
+          @subscriptions.last.connect if @started
```

Comment:
I think we could do a method that is a verbose check rather than being reliant on `@started`. 
```
def started?
  @started == true
end
```
This also helps if we change how we do the check around the check for started processes.